simulator/risc_v/t/unit_test.cpp

```diff
@@ -824,3 +824,22 @@ TEST_CASE("RISCV RV32 minu")
         cases[i].make_test("minu");
     }
 }
+
+TEST_CASE("RISCV RV32 packu")
+{
+    CHECK( RISCVInstr<uint32>(0x244D60533).get_disasm() == "packu $a0, $a2, $a3");
```

Comment:
This constant is wider than uint32. Please fix.

Reply:
```
../risc_v/t/unit_test.cpp:830:31: error: unsigned conversion from ‘long int’ to ‘uint32’ {aka ‘unsigned int’} changes value from ‘9744811315’ to ‘1154876723’ [-Werror=overflow]
  830 |     CHECK( RISCVInstr<uint32>(0x244D60533).get_disasm() == "packu $a0, $a2, $a3");
```

Reply:
I am in trioble with it. I can't get how translate string to constant in correct way. Is there any guides?

Reply:
I do not understand what string you are trying to convert to constant and why.
The error here is that your constant (0x244D60533) has 34 bits, while it should be 32 bits.

Reply:
As far as i understand i need to convert 0100100$a3$a2100$a00110011 to uint32. Am i wrong?

Reply:
Yes. It is a constant in binary form. I assume you can convert it to hexadecimal form.